lib/galaxy/tools/parser/interface.py

```diff
@@ -5,6 +5,8 @@
 
 import six
 
+from galaxy.jobs.error_level import StdioErrorLevel
```

Comment:
I was discussing that with @jmchilton yesterday, and we didn't want to import from galaxy.jobs to keep this separate. We could move the error_level code into tools though ?

Reply:
@mvdbeek Sure, that sounds good to me -- happy to move it wherever, I just want to use the constant.

Reply:
Pushed a fix for this, the refactoring I was hoping for actually eliminates some need for delayed imports and such so we closer to untangling circular imports in Galaxy I think. Thanks for working on this all!